data/common.yaml

```diff
@@ -1 +1,16 @@
---- {}
+---
+
+profile_gpu::dcgm::install::bind_mount_install:        # See reference.md for when this is needed, set to undef on purpose
+profile_gpu::dcgm::install::bind_dst_path: ""          # See reference.md for when this is needed
+profile_gpu::dcgm::install::bind_mnt_options: "nosuid,nodev,noatime,bind"
+profile_gpu::dcgm::install::bind_parent_dst_mount: ""  # See reference.md for when this is needed
+profile_gpu::dcgm::install::bind_src_path: ""          # See reference.md for when this is needed
+
+profile_gpu::dcgm::install::install_dcgm: true
+profile_gpu::dcgm::install::packages:
+  - "datacenter-gpu-manager"
+profile_gpu::dcgm::install::dcgm_version: "2.4.6-1"  # 2.4.6-1 is known to work, 3.0.4-1 has problems
```

Comment:
Should these instead be under `data/os/RedHat.yaml`?

While we only support RHEL at this point, but this seems better long term just in case.

Reply:
We can leave as is and address if this becomes needed later.